client/my-sites/plugins/education-footer/index.tsx

```diff
@@ -155,15 +156,27 @@ const EducationFooter = () => {
 	const { __ } = useI18n();
 	const localizeUrl = useLocalizeUrl();
 	const dispatch = useDispatch();
+	const { openArticleInHelpCenter } = useOpenArticleInHelpCenter();
+	const isLoggedIn = useSelector( isUserLoggedIn );
 
 	const onClickLinkCard = useCallback(
-		( content_type: string ) => {
+		( content_type: string, url: string ) => ( e: React.MouseEvent< HTMLAnchorElement > ) => {
 			dispatch(
 				recordTracksEvent( 'calypso_plugins_educational_content_click', { content_type } )
 			);
+			if ( isLoggedIn ) {
+				e.preventDefault();
+				openArticleInHelpCenter( url );
+			}
 		},
-		[ dispatch ]
+		[ dispatch, openArticleInHelpCenter ]
+	);
+
+	const websiteBuildingLink = localizeUrl( 'https://wordpress.com/support/plugins/' );
```

Comment:
Can these be localized as an object instead?

```
const links = localizeUrl({
        websiteBuilding: 'https://wordpress.com/support/plugins/',
        customization: 'https://wordpress.com/support/plugins/install-a-plugin/',
        seo: 'https://wordpress.com/support/plugins/find-and-choose-plugins/'
    });
```

And then referenced like `links.websiteBuilding`.

Reply:
I dig it! Added in [683aaa0](https://github.com/Automattic/wp-calypso/pull/94501/commits/683aaa03a96509837a2774294e7c4b4ce3425586).

Reply:
I'm not seeing these type errors locally. But it could be me. 😅 Unfortunately, I get an error using the suggestion.
<img width="786" alt="Screenshot 2024-09-16 at 11 45 46 AM" src="https://github.com/user-attachments/assets/0ccbb099-ebfd-44ee-a361-6fe1610358c8">

Reply:
That's weird! Maybe I'm out of sync.